sig-network/concepts-and-designs/kubesphere-network-policy.md

```diff
@@ -11,7 +11,7 @@ Now kubernetes defines NetworkPolicy CRD, we can use it to achieve network isola
 There two kinds of isolation i.e., workspace and project. Workspace uses a new field "networkIsolation" in spec, and project uses namespace annotation "kubesphere.io/network-isolate = enabled". The two types of isolation, which are different from the design of K8s network policy, cannot coexist, so we can just
```

Comment:
The two types of isolation, which are different from the design of K8s network policy, cannot coexist, so we can just
=> The two types of isolation are different from the design of K8s network policy, they cannot coexist in a cluster, so we can just choose one of them to achieve the isolation.